eventlet/support/greendns.py

```diff
@@ -179,11 +179,16 @@ def _readlines(self):
         Note that this performs disk I/O so can be blocking.
         """
         lines = []
+        if sys.version_info < (3, 4):
+            mode = 'rU'
+        else:
+            mode = 'r'
```

Comment:
The reason for the version check is, earlier versions have 'universal newlines' off by default, while newer versions have it on by default and give a `DeprecationWarning` if the 'U' flag is passed anyway.